python/lsst/pipe/tasks/ingestCalibs.py

```diff
@@ -39,6 +39,29 @@ def getCalibType(self, filename):
             obstype = "fringe"
         return obstype
 
+    def getDestination(self, butler, info, filename):
+        """Get destination for the file
+
+        @param butler      Data butler
+        @param info        File properties, used as dataId for the butler
+        @param filename    Input filename
+        @return Destination filename
+        """
+        #raw = butler.get("raw_filename", info)[0]  # this definitely won't work
+        #info["ccdnum"] = 1  # this is a decam-only hack for CP products
```

Comment:
This pull request is very much a placeholder at present; I will definitely test as you suggested on calibs that have been created with pipe_base and clean up commented-out lines etc.